### DISCUSSION TITLE
Lesson 5: TypeError while running deploy.py script in brownie

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1339] [Q&A]
I initialized a new brownie project via the terminal by
`$ brownie init`

I wrote the following script in python:
```
def simple_print():
    print("Hello World!")

def main():
    simple_print()
```
saved it as deploy.py in the srcipts folder and then ran the following command in the terminal:

`$ brownie run scripts/deploy.py`

and got the following error:
```
INFO: Could not find files for the given pattern(s).
Brownie v1.18.1 - Python development framework for Ethereum

BrownieSimpleStorageProject is the active project.

Launching 'ganache-cli.cmd --chain.vmErrorsOnRPCResponse true --server.port 8545 --miner.blockGasLimit 12000000 --wallet.totalAccounts 10 --hardfork istanbul --wallet.mnemonic brownie'...
  File "C:\Users\kashy\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\_cli\run.py", line 51, in main
    return_value, frame = run(
  File "C:\Users\kashy\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 53, in run
    module = _import_from_path(script)
  File "C:\Users\kashy\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 149, in _import_from_path
    _import_cache[import_str] = importlib.import_module(import_str)
  File "C:\Users\kashy\AppData\Local\Programs\Python\Python39\lib\importlib\__init__.py", line 122, in import_module
    raise TypeError(msg.format(name))
TypeError: the 'package' argument is required to perform a relative import for '.Works..CS.Web3.brownie_simple_storage.scripts.deploy'
Terminating local RPC client...
```

Been at this for a couple days now, cannot find a solution. Please advise.

**Comment by cromewar** [comment]
Hello @KashyapPanchal please refer to #980 in order to know how to make questions on the repo.
Please share the code you are using to deploy.

**Reply by ghost** [reply]
Updated the issue. Please advise. Haven't been able to find a solution for 4 days now. @PatrickAlphaC

**Reply by cromewar** [reply]
For me it seems to be a problem with your brownie installation and python environment so please take in consideration this recommendations:
1. Uninstall brownie trough pipx.
2. Create a new python virtual environment (preferably with python version 3.8 oer 3.9)
3. On the new environment install brownie (eth-brownie) with regular pip (not pipx)
4. Don't forget to select the python interpreter ( your virtualenv) on vscode.

I hope this info might help.
